asapdiscovery-ml/asapdiscovery/ml/cli.py

```diff
@@ -1285,6 +1302,7 @@ def build_and_train_schnet(
 @overwrite_args
 def build_and_train_e3nn(
     output_dir: Path | None = None,
+    weights_path: Path | None = None,
```

Comment:
Need to pass these args further down, ie push weights path into whatever is supposed to load it, apologies if premature review.

Reply:
Could you clarify this? Do you mean I should switch the sequence of these decorators? Thanks

Reply:
I thought weights path wasn't being passed to the classes, but I was wrong, my bad, ignore me.